### DISCUSSION TITLE
Contribution graph

### DISCUSSION THREAD

**Discussion by bracketloader** [community/community#61421] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I deleted several of my forked repos but my contribution in the graph also dipped they were 36 before, but after deleting some repos it went to 31 why is my contribution gone?

**Comment by nirav-gajera** [comment]
deleting forked repositories will affect your contribution graph on GitHub. This is because GitHub's contribution graph is calculated based on the number of commits you've made to all of your repositories, including forks. So, when you delete a forked repository, the commits you made to that repository will no longer be counted towards your contribution total.

In your case, you deleted several forked repositories, which resulted in a dip in your contribution graph from 36 to 31. The exact number of commits that were removed from your contribution total will depend on how many commits you made to the deleted repositories.

If you're concerned about your contribution graph, you can always restore the deleted repositories. However, keep in mind that this will also restore the commits that were made to those repositories, which could affect your contribution total in the future.
